db/migrations/20231109235138_misc_updates.php

```diff
@@ -0,0 +1,20 @@
+<?php
+declare(strict_types=1);
+
+use Phinx\Migration\AbstractMigration;
+
+final class MiscUpdates extends AbstractMigration
+{
+    public function up(): void
+    {
+        $this->execute("
+            ALTER TABLE `users` ADD COLUMN `blocked_notifications` TEXT NOT NULL;
```

Comment:
starting to think we should really split a lot of the fields from users table into separate tables, but not necessarily something to do in this PR

Reply:
I agree. I'll throw that in with characters.